src/webgpu/util/conversion.ts

```diff
@@ -179,6 +181,22 @@ export function floatBitsToNumber(bits: number, fmt: FloatFormat): number {
   return numberWithWrongBias * 2 ** (kFloat32Format.bias - fmt.bias);
 }
 
+/**
+ * Convert ufloat9e5 bits to a JS number
+ *
+ * The difference between `floatBitsToNumber` and `ufloatBitsToNumber`
+ * is
+ *
+ * floatBitsToNumber  = 2^(exponent - bias) * (1 + mantissa / 2 ^ numMantissaBits)
+ * ufloatBitsToNumber = 2^(exponent - bias - numMantissaBits) * mantissa
```

Comment:
This more clearly shows the difference between the two
```suggestion
 * The difference between `floatBitsToNumber` and `ufloatBitsToNumber`
 * is that the latter doesn't use an implicit leading bit:
 *
 * floatBitsToNumber  = 2^(exponent - bias) * (1 + mantissa / 2 ^ numMantissaBits)
 * ufloatBitsToNumber = 2^(exponent - bias) * (mantissa / 2 ^ numMantissaBits)
 *                    = 2^(exponent - bias - numMantissaBits) * mantissa
```

But more importantly I believe this is different from the other ufloat format, rg11b10ufloat. So the name of this function should be more specific. Since this format is quite special it probably would make sense to just specialize it to remove the `fmt` (which doesn't contain enough information to fully decode the number anyway) and rename it to be ufloat9e5-specific.

Reply:
I renamed it and added your comment edit. I'm not sure what you meant by "remove the `fmt`"